Modules/_multiprocessing/posixshmem.c

```diff
@@ -62,6 +62,52 @@ _posixshmem_shm_open_impl(PyObject *module, PyObject *path, int flags,
 }
 #endif /* HAVE_SHM_OPEN */
 
+#ifdef HAVE_SHM_RENAME
+/*[clinic input]
+_posixshmem.shm_rename
+    path_from: unicode
+    path_to: unicode
+    flags: int
```

Comment:
```suggestion
    flags: int
    /
```
I think it was mistake to make parameters keyword-or-positional in other functions. It is better to make them positional-only, it will help if we decide to rename parameters.